### PR TITLE
update pantry with add ingredient functionality

### PR DESCRIPTION
**Pull request by wdmcculloch**

### CODE REVIEW

File: src/component/foodcardlist/pantry/PantryStart.jsx [hunk 1]
```diff
@@ -22,57 +23,90 @@ const data = [
...
     setFilter(e.target.value.toLowerCase());
   }
 
+  const handleDelete = (e) => {
+    e.preventDefault();
+    let index = e.currentTarget.value;
+    let hold = [...ingredients];
+    let name = hold[index].name;
+
+    //if selected remove from select
```
Comment (aleceche) [inline src/component/foodcardlist/pantry/PantryStart.jsx]: so many commented out lines

**Review by JacobMelnick — COMMENTED** [review]
dont forget to delete the logs

File: src/component/foodcardlist/pantry/PantryStart.jsx [hunk 2]
```diff
@@ -22,57 +23,90 @@ const data = [
...
+      console.log('add', select)
+    } else {
+      // delete obj[name]
+      // setSelect(obj);
+      setSelect(select.filter(item => item !== name));
+      console.log('delete', select);
+    }
   }
 
   const handleSubmit = (e) => {
```
Comment (aleceche) [inline src/component/foodcardlist/pantry/PantryStart.jsx:69]: is this just for dev purposes? The user generally won't be looking at the console.

File: src/component/foodcardlist/pantry/PantryStart.jsx [hunk 3]
```diff
@@ -22,57 +23,90 @@ const data = [
   {
     "name": "Lemons",
     "image": lemonImg,
-    "selected": false
+    "select": false
```
Comment (aleceche) [inline src/component/foodcardlist/pantry/PantryStart.jsx]: go away...read some books

File: src/component/foodcardlist/pantry/PantryStart.jsx [hunk 4]
```diff
@@ -22,57 +23,90 @@ const data = [
...
   },
   {
     "name": "Cheese",
     "image": cheeseImg,
-    "selected": false
+    "select": false
```
Comment (aleceche) [inline src/component/foodcardlist/pantry/PantryStart.jsx]: ditto

**Review by aleceche — APPROVED** [review]
I see a lot of comments in here. Are they for future use or can we get rid of them?

**Pull request merged**